### PR TITLE
Adding Billing API

### PR DESCRIPTION
**Pull request by remleinpiotr**

### CODE REVIEW

File: livechat/tests/test_billing_api_client.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    ''' Test if client supports HTTP/1.1 protocol. '''
+    client = BillingApi.get_client(token='test')
+    assert client.create_direct_charge().http_version == 'HTTP/1.1'
+
+
+# At this point billing-api v1 support http2 but will always negotiate to use 'HTTP/1.1'
+# def test_client_supports_http_2():
+#     ''' Test if client supports HTTP/2 protocol. '''
+#     client = BillingApi.get_client(token='test', http2=True)
+#     assert client.create_direct_charge().http_version == 'HTTP/2'
```
Comment (zuczkows) [inline livechat/tests/test_billing_api_client.py:92]: Not sure what it means but don't leave commented code. Delete these lines if YAGNI.

**Review by zuczkows — CHANGES_REQUESTED** [review]

File: livechat/billing/api/v1.py [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+                                headers=headers)
+
+    def list_recurrent_charges(self,
+                               page: int = None,
+                               status: str = None,
+                               order_client_id: str = None,
+                               params: dict = None,
+                               headers: dict = None) -> httpx.Response:
+        ''' Lists all recurrent charges.
+            Args:
```
[the remark this answers is not in the document]
Comment (remleinpiotr) [reply livechat/billing/api/v1.py:238]: It really exist and Olek already know that at this point it is not added in docs.

File: livechat/billing/api/v1.py [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+            params = prepare_payload(locals())
+        return self.session.get(f'{self.api_url}/recurrent_charge',
+                                params=params,
+                                headers=headers)
+
+    def accept_recurrent_charge(self,
+                                charge_id: str,
+                                payload: dict = None,
+                                headers: dict = None) -> httpx.Response:
+        ''' Accepets specific recurrent charge.
```
Comment (zuczkows) [inline livechat/billing/api/v1.py:261]:
```suggestion
        ''' Accepts specific recurrent charge.
```

**Review by zuczkows — DISMISSED** [review]
Approved but correct one typo

**Review by zuczkows — DISMISSED** [review]

**Review by zuczkows — APPROVED** [review]

**Pull request merged**